rules-tests/CodeQuality/Rector/Class_/InlineConstructorDefaultToPropertyRector/Fixture/skip_joined_property.php.inc

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
fixture file name should not start with `skip_`, as it not skipped.

Reply:
Good catch :+1: